### ISSUE TITLE
Using plunker causes my hard drive to spinup

### ISSUE THREAD

**Issue by EdwardsDamian** [gordonmzhu/questions#272]
### What is the associated Watch and Code lesson? 

Write your response here (required).

### Include a live demo link so that I can run your code. 

Write your response here (required).

### Explain the exact steps I need to take to replicate the problem. 

Write your response here (required).

### What did you expect to happen? 

Write your response here (required).

### What actually happened? 

Write your response here (required).

### Describe what you've done so far to troubleshoot the problem. 

Write your response here (required).

**Comment by EdwardsDamian** [comment]
Not certain is there's an association or not but...when viewing watch-and-code lessons and using plunker (simultaneously) my hard drive tends to spinup and the body of my mac gets very hot.  

I can do both (watch and code & use plunker) by using an application called Spectacle that allows me to split screen.  

As near as I can tell the spinup and heating effect occurs when go to inspect code in plunkerPreviewTarget.  My expectation is that the hard drive should not spinup and the mac should not heat up.  

I'm just wondering if this is a-typical or common to those who use Plunker (?).

I tried googling plunker and hard drive spinups.  This behavior has never happened before (for example when using repl.it) and I'm new to using Plunker.  (The only other change is that I've recently upgraded my system software to Mojave 10.14.6)
